python/lsst/sims/catUtils/sersicUtils.py

```diff
@@ -1,100 +0,0 @@
-from scipy.special import gammainc
```

Comment:
I have no idea what this script was supposed to do.  It was about three years old and un-tested (so I'm going to assume it doesn't run).  I have taken this opportunity to `rm` all such code (though I tried to make each `rm` its own commit, in case someone really wants it back).